app/renderer/components/main/clearBrowsingDataPanel.js

```diff
@@ -138,7 +138,7 @@ class ClearBrowsingDataPanel extends React.Component {
             checkedOn={this.props.savedSiteSettings}
             onClick={this.onToggleSavedSiteSettings} />
           <SwitchControl
-            rightl10nId='Brave ads'
+            rightl10nId='braveAds'
```

Comment:
This shouldn't be changed

Reply:
Under 'Clear Browsing Data', should 'Brave ads' not be 'Brave Ads'?

Reply:
Yes:
https://github.com/brave/browser-laptop/blob/usermodel/app/extensions/brave/locales/en-US/preferences.properties#L75